src/poliastro/coordinates.py

```diff
@@ -130,3 +131,36 @@ def inertial_body_centered_to_pqw(r, v, source_body):
     v_pqw = (np.array([-sin(nu), (ecc + cos(nu)), 0]) * sqrt(k / p)).T * u.km / u.s
 
     return r_pqw, v_pqw
+
+
+def change_frame(orbit, frame_orig, frame_dest):
+    """Transforms Orbit from one frame to another.
+    
+    Parameters
+    ----------
+    orbit : ~poliastro.bodies.Orbit
+        Orbit to transform
+    frame_orig : ~astropy.coordinates.BaseRADecFrame
```

Comment:
This is in fact `BaseCoordinateFrame` http://docs.astropy.org/en/stable/api/astropy.coordinates.BaseCoordinateFrame.html

Reply:
done

Reply:
I still see it there?